packages/client/src/components/app/SidePanel.svelte

```diff
@@ -51,6 +64,7 @@
       } else {
         if (target.contains(node)) {
           target.removeChild(node)
+          handleSidePanelClose()
```

Comment:
I'm not sure what the default svelte behaviour is here, but I would just double check that this won't invoke the close callback immediately on first render, when the side panel is closed by default.